rtos/rtos/Thread.h

```diff
@@ -275,11 +276,13 @@ class Thread {
                      osPriority priority=osPriorityNormal,
                      uint32_t stack_size=DEFAULT_STACK_SIZE,
                      unsigned char *stack_pointer=NULL);
+    static void _thunk(const void * thread_ptr);
```

Comment:
It accesses private members which is why I didn't make it static to the cpp file.